src/transformState.js

```diff
@@ -5,7 +5,32 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (let i = 0; i < actions.length; i++) {
+    const { type, extraData, keysToRemove } = actions[i];
+
+    switch (type) {
+      case 'addProperties':
```

Comment:
Move all cases to the object.

Example of usage:
`case ActionType.AddProperties:`